card/mod/core/set/all/haml.rb

```diff
@@ -2,22 +2,22 @@
 #set_scope = "haml"
 
 format do
+
+  include Card::Set::Format::HamlViews
+
   define_method :the_scope do
     set_scope
   end
 
   define_method :haml_scope do
     set_scope
   end
-end
 
-format do
   # Renders haml templates. The haml template can be passed as string or
   # block or a symbol that refers to a view template.
   # @param  template_or_locals [Hash, String, Symbol]
   #   If a symbol is given then a template is expected in the corresponding view
-  #   directory. Note that {view_template_path} needs to be overridden in
-  #   that case to get the right path to the template.
+  #   directory.
   # @return [String] rendered haml as HTML
```

Comment:
I need to document the new pattern, which in short is this:
1. `haml` replaces `render_haml`
2. `haml_partial` replaces `render_haml_partial`
3. for `haml` the template path comes from the caller by default
4. for `haml_partial` (which should mostly be used within haml), the template path comes from the haml call that  called the partial
5. in either case, a `:template_path` argument may be sent in the locals to override the default path.

Reply:
but I want to get your approval of that pattern first :)